management_service/app/routers/orders.router.ts

```diff
@@ -28,13 +29,77 @@ ordersRouter.get("/:id", (req: Request, res: Response) => {
         ],
         status: "In progress",
         total: 15.99,
-        timePlaced: "2023-10-01T12:00:00Z",
+        timePlaced: new Date().toISOString(),
     }
 
     res.status(200).json({
-        message: "Order retrieved successfully",
         order: sampleOrder,
     });
+    } catch (error) {
+        console.error("Error retrieving order:", error);
+         res.status(500).json({ error: "Internal server error" });
+        
+    }
+    
+});
+
+// MIGHT REPLACE THIS WITH WEBSOCKET
+ordersRouter.post("/create-order", (req: Request, res: Response) => {
```

Comment:
A post request to the root with no id should be the create. `/create-order` is RPC over HTTP, not REST. Replace this with a post to `/`.